papr/jjb/Jenkinsfile

```diff
@@ -0,0 +1,17 @@
+// XXX: extend openshift/jenkins S2I builder so we can easily bake custom cloud
+// configs in config.xml.tpl so we don't need to `podTemplate` each time
+podTemplate(label: 'papr-slave', cloud: 'openshift', containers: [
+    // XXX: directly using the clusterIP is the only way I could get this to
+    // work -- using 'docker-registry.default.svc.cluster.local' fails
```

Comment:
Hm...

Reply:
Yeah, that annoyed me! I'd like to get back to this eventually, though looking at the documentation, it seems like that's just how it goes, e.g.: https://docs.openshift.com/enterprise/3.1/install_config/install/docker_registry.html#maintaining-the-registry-ip-address

> If you are unable to re-use the IP address, any operation that uses a pull specification that includes the old IP address will fail.